### DISCUSSION TITLE
No sources found

### DISCUSSION THREAD

**Discussion by ghost** [kitadai31/revanced-patches-android6-7#4] [General]
Once I changed the source in the settings of Revanced Manager to what you've been saying, Revanced Manager doesn't find any patches at all. What do I do wrong?
I'm using Android 6.01.

**Comment by kitadai31** [comment]
The github server was down at that time.
https://www.githubstatus.com/history
\> Apr 18, 14:35 - 15:29 UTC

This is probably the reason why you could not download the patch.
Please try again and let me know the result.
